src/components/Main.tsx

```diff
@@ -243,13 +244,17 @@ export const Main = () => {
     }
   }, [theme])
 
+  const context = useSelector(state => state.core.context)
+  const localUsers = useWatch(context, 'localUsers')
```

Comment:
Why are we watching it if we want a one-off value (hence the eslint ignore you added below)? Could it be that the localUsers is not initialized with the final state we wish to track? If so, then your eslint ignore wont ever see it.